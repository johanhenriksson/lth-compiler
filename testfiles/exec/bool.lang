bool main() {
    bool b = true;
    return false;
}
