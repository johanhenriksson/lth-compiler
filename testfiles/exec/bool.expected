; Library Functions
declare void @print(i64)
declare i64 @read()
declare i64 @truncate(double)
declare double @promote(i64)
declare void @assert(i1)
declare void @except(i1)

define i1 @main() {
%b = alloca i1
store i1 1, i1* %b
ret i1 0
}

